src/js/MediaTrackInfo.js

```diff
@@ -45,15 +45,17 @@ export default class MediaTrackInfo extends React.Component {
             if (this.props.countDirection === "COUNTDOWN") {
                 var startPosition = new Date(startDate.getTime() - offset)
                 timeSince = new Date((now - this.props.updateTime) * this.props.countRate)
-                var position = new Date(startPosition - timeSince)
-
+                // Start position is used if paused
+                var position = this.props.paused ? startPosition : new Date(startPosition - timeSince)
                 // Clamp position to timer bounds
                 position = endDate && position < endDate ? endDate : position
                 var endTime = ""
             }
             else {
                 startPosition = new Date(startDate.getTime() + offset)
                 timeSince = new Date(startPosition.getTime() + ((now - this.props.updateTime) * this.props.countRate))
```

Comment:
```suggestion
                timeSince = new Date((now - this.props.updateTime) * this.props.countRate)
```
This change is needed as well

Reply:
In fact, with this change, `timeSince` and `startPosition` can be moved out of this `if` structure